### PR TITLE
building with w3emc

### LINKED ISSUE #59: Change dependency to w3emc
Change dependency to w3emc.

### PR DESCRIPTION
**Pull request by edwardhartnett**
Fixes #59 

Now build with w3nco instead of w3emc.

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -33,10 +33,10 @@ find_package(PNG REQUIRED)
 find_package(ZLIB REQUIRED)
 
 # Find the NCEPLIBS libraries we need.
+find_package(bacio 2.4.0 REQUIRED)
```
Comment (BrianCurtis-NOAA) [inline CMakeLists.txt:36]: Should this be 2.4.1 ?

Comment (edwardhartnett) [reply CMakeLists.txt:36]: No, 2.4.0 is correct.

This number represents the minimum version we require, not the preferred version. So whatever version is provided, it must be 2.4.0 or better.

So we only want to change this number when something changes in the bacio library, and that causes incompatibilities in w3emc for older versions. We should prefer to build with the last few versions of a package at least, if we can...

**Comment by Hang-Lei-NOAA** [comment]
I agree with the version number increase.

File: CMakeLists.txt (hunk 1 above)
Comment (BrianCurtis-NOAA) [reply CMakeLists.txt:36]: OK Thanks.

**Review by BrianCurtis-NOAA — APPROVED** [review]

**Pull request merged**